.github/workflows/reusable_redo_typo_pr.yml

```diff
@@ -0,0 +1,45 @@
+name: Redo Typo PR
+
+on:
+  workflow_dispatch:
```

Comment:
this needs to be `workflow_call` if this is supposed to be a reuseable action. or we can keep the dispatch but still need to add the `_call` option 

https://github.com/celestiaorg/.github/blob/2f1460fdef3e738bc089d1c6f1bc2828f8479783/.github/workflows/reusable_dockerfile_lint.yml#L4